tasks/download/maven_repository.yml

```diff
@@ -1,8 +1,16 @@
 ---
+# install python3 variant of lxml for >3.5, which is the lowest python3 version supported by ansible
+- name: Install python3-lxml dependency for Ansible maven_artifact module.
+  package:
+    name: python3-lxml
+    state: present
+  when: ansible_python_version is version_compare('3.5', operator='>=')
```

Comment:
`version_compare` is deprecated, it's just `version` now (see [here](https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison)).

Also, I'd check for >= 3.0 here to not have that strange version gap between 3.0 and 3.5 where nothing gets installed. I wouldn't implicitly encode the Ansible Python3 support policy here, it's kind of arbitrary anyway and might change in the future.

Reply:
I kept `version_compare`, because it works with the newer versions of ansible, but `version` does not not work with older versions. Using different tasks depending on the ansible version seemed exaggerated to me, especially when the command to compare version range can't be used.